test/functional/ui/screen_advanced_spec.lua

```diff
@@ -0,0 +1,79 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear, feed, insert = helpers.clear, helpers.feed, helpers.insert
+local execute = helpers.execute
+
+
+describe('Screen rendering', function()
+  local screen
+  local colors = Screen.colors
+  local hl_colors = {
```

Comment:
I would suggest removing this and just use  `Screen.colors` directly (it is an anti-pattern from the pre-`snapshot_util()`-days that exist in a few of our existing screen tests. I will do a clean-up PR soon to remove it from existing tests.).